ci/slab.toml

```diff
@@ -85,5 +85,5 @@ check_run_name = "CSPRNG randomness testing"
 
 [command.code_coverage]
 workflow = "code_coverage.yml"
-profile = "cpu-small"
+profile = "cpu-big"
```

Comment:
what do we gain with the big instance for coverage ? I thought it was monothreaded ?

btw idea : how about launching some coverage in parallel by cloning the repo twice ?

Reply:
Yeah you right, I've to come back to `cpu-small` profile.
Regarding the launching in parallel, I'm not sure it can work unless you parametrized the workflow and run one coverage on one AWS machine using a matrix, in the same fashion than start_full_benchmarks.yml`

Reply:
well you could use gnu parallel I believe ? launching one from tfhe-rs-boolean and one from tfhe-rs-shortint

Reply:
I'll try to implement usage of GNU Parallel. Stay tuned.

Reply:
Seems to work, I'll give it a try on the CI to be sure that we reduce the total duration.

Reply:
By the way I rolled back to `cpu-big` profile in ordrer to be sure to be able to handle all the keys in memory, once we got all the layers running coverage in parallel.